src/styles/global.css

```
@@ -358,3 +422,556 @@
+
+/* Baseline letter spacing — always on in Spirit Scroll, never shifts on hover */
+[data-theme="new"] .data-table tbody td {
+  letter-spacing: 0.06em;
+}
+
+/* Divina — blue spirit */
+.dark[data-theme="new"] .data-table tbody tr[data-attribute="divina"]:hover td:not([data-column="bonds"]):not([data-column="sources"]):not([data-column="skill"]):not([data-column="ability1"]):not([data-column="ability2"]) {
+  --anchor: rgba(180, 210, 255, 0.95);
+  --halo: rgba(80, 130, 255, 0.8);
+  --fog: rgba(30, 70, 220, 0.5);
+  animation: respiration 5s ease-in-out infinite;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)